docs/understanding-your-pipeline/data-products/index.md

```diff
@@ -72,6 +72,32 @@ Break down the barriers that exist between data producers and data consumers, by
 
 ![Key elements of a Data Product](images/key-elements-of-a-data-product.png)
 
+**Volume metrics**; data products can identify events ingested in your pipeline that match the configured event specifications. This allows your team to visualize the volume-related metrics for events being tracked to specific application IDs.
+
+**Benefits:**
+
+You will be able to view several items in the UI that help identify and detect anomalies or potential misconfigurations in trackers that are not sending the expected events. This is particularly useful during the development phase when implementing tracking for a specific application using [Snowtype](/docs/collecting-data/code-generation). These visual elements include:
+
+* A counter for each event specification, showing the total number of events detected from the tracked [applications IDs](/docs/understanding-tracking-design/organize-data-sources-with-source-applications/index.md#application-ids) in the last 30 days.
+* A 'last seen' field for each event specification, indicating when the last event matching the event specification was detected.
+* A list of application IDs from which events are being tracked, displayed for each event specification. For each application ID, a status will be shown with different colors:
+  * __Green__: Events conforming to the event specification are being tracked and identified with the specific application ID inherited from the configured [source applications](/docs/understanding-tracking-design/organize-data-sources-with-source-applications/index.md).
+  * __Gray__: No events are being tracked for an application ID inherited from the configured source applications.
+  * __Yellow__: Events are being tracked for an application ID that has not been configured or inherited from the source applications.
+
+:::note
+
+Some data products, such as [__Base Web__](/docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md#base-web) and [__Base Mobile__](/docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md#base-mobile), contain standard events (e.g., _page pings_, _link clicks_, _screen view_, _application install_). 
+
+For these data products, the volume metrics will, by default, display the volume of all events associated with the application IDs for the specific data product, similar to other non-base data products.
```

Comment:
These data products, when associated with a Source Application, will display metrics for the standard events without requiring tracking these particular events through Snowtype.

Reply:
Does that mean that the toggle will be active/inactive by default depending on if the data product has any source application associated? I find this confusing. 🤔

Reply:
The toggle will be always off in the initial panel.

If the Base Data Product has a Source Application associated, switching it on will show you other tracked appIds. (_Soon you will not be able to create a base Data Product without a Source Application in the UI_)

If it does not have any Source App associated, will do the exact same thing but by default you don't see any metrics being counted.

The toggle is not rendered If there are no other appIds tracked for these base events.

Reply:
@igneel64  I attempted to phrase all that. Let me know if it makes sense